object/slicer/options.go

```diff
@@ -45,6 +47,13 @@ func (x *Options) SetBearerToken(bearerToken *bearer.Token) {
 	x.bearerToken = bearerToken
 }
 
+// SetCopiesNumber sets the number of object copies.
```

Comment:
i think it confuses a little: sounds like it overrides `REP` in policy

Reply:
Variants?

Reply:
Updated